-------------------------------------------------------------------------------
Test set: market.v3_gpt4turbo_run03_RestErrorResponseTest
-------------------------------------------------------------------------------
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.809 s <<< FAILURE! - in market.v3_gpt4turbo_run03_RestErrorResponseTest
testValidResponse_for500InternalError_whenFieldError  Time elapsed: 0.091 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at market.v3_gpt4turbo_run03_RestErrorResponseTest.testValidResponse_for500InternalError_whenFieldError(v3_gpt4turbo_run03_RestErrorResponseTest.java:72)

